src/vl-map.js

```diff
@@ -130,13 +149,13 @@ export class VlMap extends vlElement(HTMLElement) {
   get _projection() {
     return new OlProjection({
       code: 'EPSG:31370',
-      extent: this._extent,
+      extent: [9928, 66928, 272072, 329072],
```

Comment:
Duplicate, hier kan beter `this.__extent` gebruikt worden.

Reply:
Neen, want dit is niet de view extent. Dit is de extent van de projectie. Heb dit aangepast naar de correcte bbox.